### DISCUSSION TITLE
[Request] Allow builders to grab multiple items from storage at once without returning to command table between each!

### DISCUSSION THREAD

**Discussion by Jax2** [ldtteam/minecolonies#7921] [Ideas]
As it is currently, the mod is exceedingly frustrating when starting new buildings. If you are using the medieval dark oak builds, the builders hut has the storage area in a separate building, in a basement. For some odd reason, the builder is limited to grabbing one item (or stack of) before having to return to the "command block", and then returning for the next item. This makes little sense to me, and it is causing MASSIVE delays on getting anything done. Why is it not possible to grab a list of blocks to be taken from storage once, then going to storage and filling that list? Why must there be a return trip to the main table between each block grab? 

I have been waiting for 35 minutes now for my builder to upgrade the builders hut from 2 -> 3 because he is constantly just going back and forth for one type of block / item at a time. With 27 slots to fill, it takes forever the way it is. If you don't manually open the doors and leave them open, then he has to pause, open the door, go to next door, pause and open that too, both directions. Hair pulling!

**Comment by sidara21** [comment]
I've removed the doors on that particular builder's hut design because it's so frustrating.  I'd second this request - depending on the hut design type (wooden is fine, from memory, racks are all on one level and close by) it can take a builder close to half a day to pick up all the things they need for the current section of the build leaving very little time for actual building.
